source/features/profile-gists-link.css

```diff
@@ -1,5 +1,4 @@
 /* Fix wrapping text on tabs when feature is enabled #1757 */
-.UnderlineNav-item:nth-last-child(n+7),
-.UnderlineNav-item:nth-last-child(n+7) ~ * {
+.UnderlineNav-item:is(:nth-last-child(n+7), :nth-last-child(n+7) ~ *) {
```

Comment:
For this selector specifically, it might be worth checking if #1757 still applies after 2 years

Reply:
I dont have a way to check. I only have 4 tabs

Reply:
It doesn't apply only to your own profile. To test it you just have to disable the "hide project tab" feature. My guess is that they added a scrollbar on that tab bar so it no longer applies.